sql/wsrep_mysqld.cc

```diff
@@ -2004,6 +2004,27 @@ static void wsrep_RSU_end(THD *thd)
   thd->variables.wsrep_on= 1;
 }
 
+
+static bool global_read_lock_set(THD *thd)
+{
+  MDL_key key;
+  THD *tmp;
+  bool ftwrl1_lock= false, ftwrl2_lock= false;
+  if (thd->global_read_lock.is_acquired()) return true;
+  key.mdl_key_init(MDL_key::BACKUP, "", "");
+  longlong id = thd->mdl_context.get_lock_owner(&key);
+  if(id && (tmp= find_thread_by_id(id)))
+  {
+    ftwrl1_lock= tmp->mdl_context.is_lock_owner(MDL_key::BACKUP, "", "",
+                                                MDL_BACKUP_FTWRL1);
+    ftwrl2_lock= tmp->mdl_context.is_lock_owner(MDL_key::BACKUP, "", "",
+                                                MDL_BACKUP_FTWRL1);
```

Comment:
What's the point having two different variables holding the same? Besides `is_lock_owner()` is only allowed to be called in the context owner thread.

Reply:
Should here MDL_BACKUP_FTWRL2 used ? How you change contex owner ?

Reply:
Double FTWRL1 was typo, fixed.